contracts/payloads/IGP100/PayloadIGP100.sol

```diff
@@ -0,0 +1,661 @@
+                    IFluidAdminDex.UserBorrowConfig[]
+                        memory config_ = new IFluidAdminDex.UserBorrowConfig[](
+                            1
+                        );
+                    config_[0] = IFluidAdminDex.UserBorrowConfig({
+                        user: wstUSR_USDC__USDC_USDT_VAULT,
+                        expandPercent: 30 * 1e2, // 20%
+                        expandDuration: 6 hours, // 12 hours
+                        baseDebtCeiling: 4_000 * 1e18, // 4k shares ($8k)
+                        maxDebtCeiling: 5_000 * 1e18 // 5k shares ($10k)
+                    });
+
+                    IFluidDex(USDC_USDT_CONCENTRATED_DEX)
+                        .updateUserBorrowConfigs(config_);
+                }
+
+                VAULT_FACTORY.setVaultAuth(
+                    wstUSR_USDC__USDC_USDT_CONCENTRATED_VAULT,
+                    TEAM_MULTISIG,
+                    true
+                );
+            }
+        }
+    }
+
+    // @notice Action 8: Set dust limits for GHO-USDe DEX and its vault
+    function action8() internal isActionSkippable(8) {
+        {
+            // GHO-USDe DEX
+            DexConfig memory DEX_GHO_USDe = DexConfig({
+                dex: GHO_USDe_DEX,
+                tokenA: GHO_ADDRESS,
+                tokenB: USDe_ADDRESS,
+                smartCollateral: true,
+                smartDebt: false,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 0, // $0
+                maxBorrowLimitInUSD: 0 // $0
+            });
+            setDexLimits(DEX_GHO_USDe); // Smart Collateral
+
+            DEX_FACTORY.setDexAuth(GHO_USDe_DEX, TEAM_MULTISIG, true);
+        }
+
+        {
+            address GHO_USDe__GHO_USDC_VAULT = getVaultAddress(136);
+            address GHO_USDC_DEX = getDexAddress(4);
+
+            {
+                // Update GHO-USDe<>GHO-USDC vault borrow shares limit
+                IFluidAdminDex.UserBorrowConfig[]
+                    memory config_ = new IFluidAdminDex.UserBorrowConfig[](1);
+                config_[0] = IFluidAdminDex.UserBorrowConfig({
+                    user: GHO_USDe__GHO_USDC_VAULT,
+                    expandPercent: 30 * 1e2, // 20%
+                    expandDuration: 6 hours, // 12 hours
+                    baseDebtCeiling: 4_000 * 1e18, // 4k shares ($8k)
+                    maxDebtCeiling: 5_000 * 1e18 // 5k shares ($10k)
+                });
+
+                IFluidDex(GHO_USDC_DEX).updateUserBorrowConfigs(config_);
+            }
+
+            VAULT_FACTORY.setVaultAuth(
+                GHO_USDe__GHO_USDC_VAULT,
+                TEAM_MULTISIG,
+                true
+            );
+        }
+    }
+
+    // @notice Action 9: Increase Borrow Limits on GHO-USDC DEX
+    function action9() internal isActionSkippable(9) {
+        address GHO_USDC_DEX = getDexAddress(4);
+        {
+            IFluidDex(GHO_USDC_DEX).updateMaxBorrowShares(
```

Comment:
leave max borrow shares as is, we need to increase limits for the dex at liquidity layer to 30M$ each 

base = max * 0.6 so set to 18M